extendr-api/src/wrapper/promise.rs

```diff
@@ -29,6 +29,7 @@ impl Promise {
         })
     }
 
+    #[cfg(feature = "non-api")]
     /// Get the code to be executed from the promise.
     pub fn code(&self) -> Robj {
```

Comment:
Should just note that like everything promise related is now non-api. 

Could this be related to the CVE in R? lol